samples/document-snippets/instance.js

```diff
@@ -0,0 +1,382 @@
+      console.log(`Table created: ${newTable.name}`);
+    })
+    .catch(err => {
+      console.error('Error creating table: ', err);
+    });
+  // [END bigtable_create_table]
+}
+
+function existsInstance(instanceId) {
+  const instance = bigtableClient.instance(instanceId);
+
+  // [START bigtable_exists_instance]
+  instance
+    .exists()
+    .then(result => {
+      const exists = result[0];
+      console.log(`Instance ${instanceId} Exists: ${exists}`);
+    })
+    .catch(err => {
+      console.error('Error in checking Instance exists: ', err);
+    });
+  // [END bigtable_exists_instance]
+}
+
+function getInstance(instanceId) {
+  const instance = bigtableClient.instance(instanceId);
+
+  // [START bigtable_get_instance]
+  instance
+    .get()
+    .then(result => {
+      const instance = result[0];
+      // const apiResponse = result[1];
+      console.log(`Instance: \n${instance}`);
+    })
+    .catch(err => {
+      console.error('Error geting Instance: ', err);
+    });
+  // [END bigtable_get_instance]
+}
+
+function getClusters(instanceId) {
+  const instance = bigtableClient.instance(instanceId);
+
+  // [START bigtable_get_clusters]
+  instance
+    .getClusters()
+    .then(result => {
+      console.log(`Clusters: \n${result[0]}`);
+    })
+    .catch(err => {
+      console.error('Error geting Clusters: ', err);
+    });
+  // [END bigtable_get_clusters]
+}
+
+function getAppProfiles(instanceId) {
+  const instance = bigtableClient.instance(instanceId);
+
+  // [START bigtable_get_app_profiles]
+  instance
+    .getAppProfiles()
+    .then(result => {
+      console.log(`AppProfiles: \n${result[0]}`);
+    })
+    .catch(err => {
+      console.error('Error geting AppProfiles: ', err);
+    });
+  // [END bigtable_get_app_profiles]
+}
+
+function getMetaData(instanceId) {
+  const instance = bigtableClient.instance(instanceId);
+
+  // [START bigtable_get_imeta]
+  instance
+    .getMetadata()
+    .then(result => {
+      const metaData = result[0];
+      // const apiResponse = result[1];
+      console.log('%s %O', 'Instance Metadata:\n', metaData);
+    })
+    .catch(err => {
+      console.error('Error geting Metadata: ', err);
+    });
+  // [END bigtable_get_imeta]
+}
+
+function getTables(instanceId) {
+  const instance = bigtableClient.instance(instanceId);
+
+  // [START bigtable_get_tables]
+
+  // To control how many API requests are made and page through the results
+  // manually, set `autoPaginate` to false.
+  const options = {
+    autoPaginate: false,
+  };
+  // const options = {
+  //   autoPaginate: true
+  // };
+
+  instance
+    .getTables(options)
+    .then(result => {
+      console.log(`Tables:`);
+      let tables = result[0];
+      tables.forEach(t => {
+        console.log(t.name);
+      });
+    })
+    .catch(err => {
+      console.error('Error geting Tables: ', err);
+    });
+  // [END bigtable_get_tables]
+}
+
+function setMetaData(instanceId) {
```

Comment:
Please call this method `updateInstance`

Reply:
Done
https://github.com/googleapis/nodejs-bigtable/pull/222/commits/174f67cadb820ec8b26349131f69e66a403fc720